Sources/CoreMetrics/Metrics.swift

```diff
@@ -448,6 +486,10 @@ public final class NOOPMetricsHandler: MetricsFactory, CounterHandler, RecorderH
         return self
     }
 
+    public func remove<M: Metric>(metric: M) {
+        // no-op
```

Comment:
Why not leave this on just one line like the other no-op's
```swift
public func remove<M: Metric>(metric: M) { }
```

Reply:
```swift
public func remove<M: Metric>(metric: M) { /* no-op */ }
```
To even make it explicit that this is intentional

Reply:
cool, will do 👍

Reply:
(not sure if a formatter like swift format won't "really want to make it 2 lines" but we'll see :))

Reply:
Done